reference/tools/pd-control.md

```diff
@@ -516,7 +487,7 @@ config set cluster-version 1.0.8
 >> hot store
 ```
 
-### label [store \<name> \<value>]
+### label [store <name> <value>]
```

Comment:
Please don't delete the `\`. Otherwise, the `<>` might not display correctly on the website.
```suggestion
### label [store \<name> \<value>]
```